src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -252,40 +252,6 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
     }
   }
 
-  test("Verify refresh-incremental (append-only) is a no-op if no new files found.") {
```

Comment:
there is another test which does exactly this in `RefreshIndexTests` so this is not required anymore. This happened after merging append and delete within refresh.